docs/design/gatway-accelerated-interface-configuration.md

```diff
@@ -0,0 +1,82 @@
+# Gateway Accelerated Interface Configuration
+
+## Description
+
+To provide hardware acceleration for traffic, both IN and OUT ports need to be a hardware 
+accelerated netdevice backed by the Network Interface Card hardware itself.
+In case of external traffic, when one such port is the external OVS bridge, which for example has the gateway IP, 
+such traffic (like host networking traffic) would not be accelerated.
+Using Switchdev VirtualFunction (VF) or SubFunction (SF) as a gateway interface allows to accelerate these too.
+
+
+## How it works?
+
+Instead of using the gateway interface as the external bridge itself, use a  switchdev VF or SF instead. 
+This is depicted as following:
+
+```
+                         +----------+
+                         |  br-ext  |
+                   +--------+       |
+                   | UPLINK |       |
+                   +--------+       |  patch  +----------+
+                         |          x---------x  br-int  |
+    +--------+     +--------+       |  port   +----------+
+    | NETDEV +-----+   REP  |       |
+    +--------+     +--------+       |
+                         +----------+
+```
+
+Where `UPLINK` is a port on an offloading capable network interface hardware, `NETDEV` is a switchdev function 
+of this port and `REP` is a representor netdevice of the switchdev function. 
+Node/Host IP assigned to `NETDEV` which make OVS to chose `REP` port for external flows instead of the bridge.
+
+
+## How to use?
+
+Gateway accelerated interface can be used in two steps:
+
+a) Creating and configuring the device. 
+See figure above.
+An `UPLINK` device is connected to the OVS external bridge. 
+An existing VF or SF `NETDEV` from the `UPLINK` is first selected as the the Gateway Interface. Its associated 
+representor `REP` is plugged into the OVS external bridge (br-ext). The gateway IP is assigned to this interface 
+instead of the OVS external bridge (br-ext). 
+
+b) Specify `NETDEV` as a gateway interface explicitly via `OVN_GATEWAY_OPTS` environment variable for
+  ovnkube-node container. Example:
+
+```yaml
+            - name: OVN_GATEWAY_OPTS
+              value: "--gateway-accelerated-interface=<<NETDEV>>"
+```
+
+Note that this is mutually exclusive to the `--gateway-interface` flag for GATEWAY_OPTIONS.
+
+c) Optionally set the external-id on the bridge to detect the uplink device.
+```bash
```

Comment:
if uplink is bond0, and bridge name is not brbond0, we must set this option.  so, it's not optional for this condition.